src/runtime/app.ts

```diff
@@ -7,6 +7,10 @@ import { generateSearchSections } from './internal/search'
 import { fetchQuery } from './internal/api'
 import { tryUseNuxtApp } from '#imports'
 
+export type {
+  ContentNavigationItem,
+}
+
```

Comment:
Can't update `queryCollectionItemSurroundings` and explicitly point to type instead of exporting type here?

```ts

import type { ContentNavigationItem } from '@nuxt/content'
export function queryCollectionItemSurroundings<T extends keyof PageCollections>(collection: T, path: string, opts?: SurroundOptions<keyof PageCollections[T]>): ChainablePromise<T, ContentNavigationItem[]> {
  return chainablePromise(collection, qb => generateItemSurround(qb, path, opts))
}
```

I guess this resolves the issue without exporting the type here. The idea is to keep all type exports in same place

Reply:
I've updated the return types of those two functions instead.